pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -294,6 +294,27 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
       "slice_start": self.start, "slice_end": self.stop, "slice_step": self.step,
       "from": self._get_input_layer_name(input)}
 
+  def _get_output_shape_from_returnn(self, inputs_flat: List[Tensor], layer: LayerBase
+                                     ) -> Tuple[Tuple[int, ...], Dict[int, int]]:
+    """
+    The size of the dynamic axes might be changed, so we have to take care of this here for the torch shape.
+    """
+    torch_shape, returnn_axis_from_torch_axis = super(Slice, self)._get_output_shape_from_returnn(
+      inputs_flat=inputs_flat, layer=layer)
+    assert len(inputs_flat) == 1
+    torch_shape = list(inputs_flat[0].shape)
```

Comment:
If you make the hard assumption here that the order of the axis will not change by this layer, then you can simply also take `returnn_axis_from_torch_axis` from the inputs, and not calculate it. Or rather, you should do this, for consistency, and also simplicity.

Reply:
Yes, done.